docs/docs/hedgehog-engine/gamelist.md

```diff
@@ -5,16 +5,21 @@ description: A list of games that are using Hedgehog Engine.
 # Game List
 
 ## Hedgehog Engine
-| Game Name                               | Internal Name |
-| --------------------------------------- | ------------- |
-| Sonic Unleashed (Sonic World Adventure) | SWA           |
-| Sonic Colors                            | sonic2010     |
-| Sonic Generations                       | BlueBlur      |
-| Sonic Lost World                        | sonic2013     |
-| Sonic Colors: Ultimate\*                | Rainbow       |
+| Game Name                                              | Internal Name |
+| ------------------------------------------------------ | ------------- |
+| Sonic Unleashed (Sonic World Adventure)                | SWA           |
+| Sonic Colors\*                                         | sonic2010     |
+| Sonic Generations                                      | BlueBlur      |
+| Sonic Lost World                                       | sonic2013     |
+| Mario & Sonic at the Sochi 2014 Olympic Winter Games\* | urals         | 
```

Comment:
We looked into Sochi 2014 before, the only thing it has in common in Hedgehog Engine is it using the lightfield code.

Reply:
okay.